HostDependencies.md

```diff
@@ -24,31 +24,66 @@ As dependencies and packages may be frequently changed, deprecated or updated th
 The basic build package on Ubuntu is the `build-essential` package. To install run:
 
 ```
-sudo apt-get update
-sudo apt-get install build-essential
+sudo apt update
+sudo apt install -y build-essential
```

Comment:
It would be best to keep using `apt-get`, since it is used in all our scripts, and furthermore:
```
The `apt` command is meant to be pleasant for end users and does not need
       to be backward compatible like apt-get(8).
```
so we may as well stick with the stable one.

I would also like to stick without `-y` so users have the option to see what is about to happen to their systems. There is a chance it could remove packages they want.

Reply:
@tecty we want to keep `apt-get` and not the `-y` option.